pkg/tls/resolve.go

```
@@ -0,0 +1,94 @@
+	ApplyToServingInfo(&config.ServingInfo, resolved)
+
+	content, err := sigsyaml.Marshal(config)
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal operator config: %w", err)
+	}
+	tmpFile, err := os.CreateTemp("", "sscsi-operator-config-*.yaml")
+	if err != nil {
+		return "", fmt.Errorf("failed to create temp config file: %w", err)
+	}
+	if _, err := tmpFile.Write(content); err != nil {
+		tmpFile.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `tmpFile.Close` is not checked

(errcheck)